test/e2e/targetallocator_resources.go

```diff
@@ -0,0 +1,109 @@
+// SPDX-FileCopyrightText: Copyright 2025 Dash0 Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e
+
+import (
+	"fmt"
+	"os/exec"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var (
+	targetAllocatorName = fmt.Sprintf(
+		"%s-opentelemetry-target-allocator-deployment",
+		operatorHelmReleaseName,
+	)
+)
+
+func deployPrometheusCrds(cleanupSteps *neccessaryCleanupSteps) {
```

Comment:
_question:_ In the test-resources script, the Prometheus CRDs are installed via `helm upgrade ... prometheus-community/prometheus-operator-crds`. What is the reason for doing it differently in the e2e tests?

Reply:
Ha, good point. I didn't remember that this is also different between scripts and e2e tests. I also don't see a real reason why it should be this way. But kudos for keeping it consistent with what is there already. I'm fine with leaving it like this for the new CRDs. Eventually we might want to align it between scripts and e2e tests both for the "new" Prometheus CRDs and also for PrometheusRule and PersesDashboard, but it's also not exactly a high priority. Let's leave it like this for now. 👍